cmake/modules/FindGMock.cmake

```diff
@@ -113,172 +117,329 @@ function(GTEST_ADD_TESTS executable extra_args)
+            set(GTEST_LIBRARY "${binary_dir}/${CMAKE_CFG_INTDIR}/${GTEST_LIBRARY_PREFIX}gtest${Suffix}")
+            mark_as_advanced(GTEST_LIBRARY)
+            set(GTEST_MAIN_LIBRARY "${binary_dir}/${CMAKE_CFG_INTDIR}/${GTEST_LIBRARY_PREFIX}gtest_main${Suffix}")
+            mark_as_advanced(GTEST_MAIN_LIBRARY)
+            externalproject_get_property(gmock source_dir binary_dir)
+            set(GMOCK_INCLUDE_DIR "${source_dir}/include")
+            mark_as_advanced(GMOCK_INCLUDE_DIR)
+            set(GMOCK_LIBRARY "${binary_dir}/${CMAKE_CFG_INTDIR}/${GTEST_LIBRARY_PREFIX}gmock${Suffix}")
+            mark_as_advanced(GMOCK_LIBRARY)
+            set(GMOCK_MAIN_LIBRARY "${binary_dir}/${CMAKE_CFG_INTDIR}/${GTEST_LIBRARY_PREFIX}gmock_main${Suffix}")
+            mark_as_advanced(GMOCK_MAIN_LIBRARY)
+        else() #1.8.0
+            externalproject_add(
+                gmock
+                GIT_REPOSITORY "https://github.com/google/googletest.git"
+                GIT_TAG "release-${GMOCK_VER}"
+                PREFIX ${GMOCK_ROOT}
+                INSTALL_COMMAND ""
+                LOG_DOWNLOAD ON
+                LOG_CONFIGURE ON
+                LOG_BUILD ON
+                CMAKE_ARGS
+                    -Dgtest_disable_pthreads=${MINGW}
+                    ${GTEST_CMAKE_ARGS}
+            )
+
+            add_dependencies(GTest::GTest gmock)
+            add_dependencies(GTest::Main gmock)
+            add_dependencies(GMock::GMock gmock)
+            add_dependencies(GMock::Main gmock)
+
+            externalproject_get_property(gmock source_dir binary_dir)
+            set(GTEST_INCLUDE_DIR "${source_dir}/googletest/include")
+            set(GTEST_LIBRARY "${binary_dir}/googlemock/gtest/${CMAKE_CFG_INTDIR}/${GTEST_LIBRARY_PREFIX}gtest${Suffix}")
+            set(GTEST_MAIN_LIBRARY "${binary_dir}/googlemock/gtest/${CMAKE_CFG_INTDIR}/${GTEST_LIBRARY_PREFIX}gtest_main${Suffix}")
+            set(GMOCK_INCLUDE_DIR "${source_dir}/googlemock/include")
+            set(GMOCK_LIBRARY "${binary_dir}/googlemock/${CMAKE_CFG_INTDIR}/${GTEST_LIBRARY_PREFIX}gmock${Suffix}")
+            set(GMOCK_MAIN_LIBRARY "${binary_dir}/googlemock/${CMAKE_CFG_INTDIR}/${GTEST_LIBRARY_PREFIX}gmock_main${Suffix}")
+            mark_as_advanced(GMOCK_LIBRARY)
+            mark_as_advanced(GMOCK_INCLUDE_DIR)
+            mark_as_advanced(GTEST_MAIN_LIBRARY)
+            mark_as_advanced(GTEST_LIBRARY)
+            mark_as_advanced(GTEST_INCLUDE_DIR)
+            mark_as_advanced(GMOCK_MAIN_LIBRARY)
+        endif()
+    else()
+        message(STATUS "Building Gmock / Gtest from dir ${GMOCK_SRC_DIR}")
+        externalproject_add(
             gmock
-            GIT_REPOSITORY "https://github.com/google/googlemock.git"
-            GIT_TAG "release-${GMOCK_VER}"
+            SOURCE_DIR ${GMOCK_SRC_DIR}
             PREFIX ${GMOCK_ROOT}
             INSTALL_COMMAND ""
-            LOG_DOWNLOAD ON
+            LOG_DOWNLOAD OFF
```

Comment:
Why don't just leave it as is? It's not displayed on the terminal anyway, but if something goes wrong you'll now have a log file to look at. This is pretty relevant in e.g. containerized builds where sometimes people mess up with the networking.